media/css/newsletter/newsletter-developer.scss

```diff
@@ -5,26 +5,40 @@
 @import '../pebbles/includes/lib';
 
 .section-intro {
-    background: url('/media/img/newsletter/developer/heart.svg') top 30px center no-repeat,
-                url('/media/img/newsletter/developer/grid.svg') top left repeat,
-                #0c99d5;
-    @include background-size(200px 199px, 60px 60px);
+    background: url('/media/img/newsletter/developer/Hearts.svg') top 30px center no-repeat,
```

Comment:
Try and keep image filenames all lowercase. This is mostly for consistency, but also as some file systems can be case sensitive, which can lead to hard-to-spot errors.